registration/templates/registration/registration_form.html

```diff
@@ -20,6 +20,9 @@
         {% include row with field=form.accept_terms only %}
         {% include row with field=form.subscribe_to_updates only %}
     {% endwith %}
+    {% with row="registration/hidden_field.html" %}
```

Comment:
@omarkhan I had included the field directly at some point and felt that it made the `registration_form.html` template kind of messy. That's the only reason; it shouldn't make a difference w/r/t functionality. If you feel strongly about it, let me know and I'll change the code to include the field directly.